awx/ui_next/src/screens/Inventory/InventoryList/InventoryList.jsx

```diff
@@ -23,225 +24,172 @@ const QS_CONFIG = getQSConfig('inventory', {
-                qsConfig={QS_CONFIG}
-                additionalControls={[
-                  <ToolbarDeleteButton
-                    key="delete"
-                    onDelete={this.handleInventoryDelete}
-                    itemsToDelete={selected}
-                    pluralizedItemName="Inventories"
-                  />,
-                  ...(canAdd ? [addButton] : []),
-                ]}
-              />
-            )}
-            renderItem={inventory => (
-              <InventoryListItem
-                key={inventory.id}
-                value={inventory.name}
-                inventory={inventory}
-                detailUrl={
-                  inventory.kind === 'smart'
-                    ? `${match.url}/smart_inventory/${inventory.id}/details`
-                    : `${match.url}/inventory/${inventory.id}/details`
-                }
-                onSelect={() => this.handleSelect(inventory)}
-                isSelected={selected.some(row => row.id === inventory.id)}
-              />
-            )}
-            emptyStateControls={canAdd && addButton}
-          />
-        </Card>
-        <AlertModal
-          isOpen={deletionError}
-          variant="error"
-          title={i18n._(t`Error!`)}
-          onClose={this.handleDeleteErrorClose}
-        >
-          {i18n._(t`Failed to delete one or more inventories.`)}
-          <ErrorDetail error={deletionError} />
-        </AlertModal>
-      </PageSection>
-    );
-  }
+      ]}
+    />
+  );
+  return (
+    <PageSection>
+      <Card>
+        <PaginatedDataList
+          contentError={contentError}
+          hasContentLoading={hasContentLoading}
+          items={inventories}
+          itemCount={itemCount}
+          pluralizedItemName={i18n._(t`Inventories`)}
+          qsConfig={QS_CONFIG}
+          onRowClick={handleSelect}
+          toolbarSearchColumns={[
+            {
+              name: i18n._(t`Name`),
+              key: 'name',
+              isDefault: true,
+            },
+            {
+              name: i18n._(t`Created By (Username)`),
+              key: 'created_by__username',
+            },
+            {
+              name: i18n._(t`Modified By (Username)`),
+              key: 'modified_by__username',
+            },
+          ]}
+          toolbarSortColumns={[
+            {
+              name: i18n._(t`Name`),
+              key: 'name',
+            },
+          ]}
+          renderToolbar={props => (
+            <DatalistToolbar
+              {...props}
+              showSelectAll
+              showExpandCollapse
+              isAllSelected={isAllSelected}
+              onSelectAll={handleSelectAll}
+              qsConfig={QS_CONFIG}
+              additionalControls={[
+                <ToolbarDeleteButton
+                  key="delete"
+                  onDelete={handleInventoryDelete}
+                  itemsToDelete={selected}
+                  pluralizedItemName="Inventories"
```

Comment:
I know this was like this before you got in here but do you mind marking this one for translation?